fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -318,17 +318,22 @@ private RdfStream getResourceTriples(final int limit, final FedoraResource resou
 
         final List<Stream<Triple>> streams = new ArrayList<>();
 
+
         if (returnPreference.getValue().equals("minimal")) {
             streams.add(getTriples(resource, of(PROPERTIES, MINIMAL)).filter(tripleFilter));
 
-            if (ldpPreferences.prefersServerManaged()) {
+            // Mementos already have the server managed properties in the PROPERTIES category
+            // since mementos are immutable and these triples are no longer managed
+            if (!resource.isMemento() && ldpPreferences.prefersServerManaged()) {
```

Comment:
might make sense to swap the order of these checks to prioritize the condition that doesn't hit the database, but its a minor thing